WebGoat-Legacy-master/src/main/java/org/owasp/webgoat/lessons/DOMInjection.java

```diff
@@ -56,11 +56,11 @@ public class DOMInjection extends LessonAdapter
 
     private final static Integer DEFAULT_RANKING = new Integer(10);
 
-    private final static String KEY = "key";
+    
 
     public final static A MAC_LOGO = new A().setHref("http://www.softwaresecured.com").addElement(new IMG("images/logos/softwaresecured.gif").setAlt("Software Secured").setBorder(0).setHspace(0).setVspace(0));
     
-    private final static String key = "K1JFWP8BSO8HI52LNPQS8F5L01N";
+    
 
     public void handleRequest(WebSession s)

```

Comment:
**Suggestion:** Empty lines have been added at lines 59 and 63. It's a good practice to remove unnecessary empty lines to keep the code clean and readable.
```suggestion
    public final static A MAC_LOGO = new A().setHref("http://www.softwaresecured.com").addElement(new IMG("images/logos/softwaresecured.gif").setAlt("Software Secured").setBorder(0).setHspace(0).setVspace(0));
    public void handleRequest(WebSession s)
```